src/Take.Blip.Builder/Actions/ProcessCommand/ProcessCommandAction.cs

```diff
@@ -43,5 +47,16 @@ public async Task ExecuteAsync(IContext context, JObject settings, CancellationT
             var resultCommandJson = _envelopeSerializer.Serialize(resultCommand);
             await context.SetVariableAsync(variable, resultCommandJson, cancellationToken);
         }
+
+        private Command ConvertToCommand(JObject settings)
+        {
+            if (settings.TryGetValue(RESOURCE_TYPE_KEY, out var type)
+                && type.ToString().Contains(SERIALIZABLE_TYPE, StringComparison.OrdinalIgnoreCase)
```

Comment:
It's not safe to just check if contains the word "json".
By the mime type nature types with JSON underlying structure should end with the suffix +json.

I think we can fill those edge cases checking if the type ends with a `/json` or `+json`